x-pack/plugins/alerting/server/task_runner/execution_handler.ts

```diff
@@ -571,15 +575,8 @@ export class ExecutionHandler<
     return executables;
   }
 
-  private canFetchSummarizedAlerts(action: RuleAction) {
-    const hasGetSummarizedAlerts = this.ruleType.getSummarizedAlerts !== undefined;
-
-    if (action.frequency?.summary && !hasGetSummarizedAlerts) {
-      this.logger.error(
-        `Skipping action "${action.id}" for rule "${this.rule.id}" because the rule type "${this.ruleType.name}" does not support alert-as-data.`
-      );
-    }
-    return hasGetSummarizedAlerts;
+  private canGetSummarizedAlerts() {
+    return this.ruleType.alerts !== undefined;
```

Comment:
can we also check here (or maybe somewhere else is more appropriate) that `this.alertsClient.getSummarizedAlerts` is defined?

Reply:
done